applications/StructuralMechanicsApplication/structural_mechanics_application_variables.cpp

```diff
@@ -30,7 +30,7 @@ KRATOS_CREATE_VARIABLE(int, SECTION_SIDES)
 KRATOS_CREATE_VARIABLE(Matrix, GEOMETRIC_STIFFNESS)
 KRATOS_CREATE_VARIABLE(Vector, LOCAL_AXIS_VECTOR_1)
 KRATOS_CREATE_VARIABLE(Matrix, LOCAL_ELEMENT_ORIENTATION)
-KRATOS_CREATE_VARIABLE(double, ORTHOTROPIC_ORIENTATION_ASSIGNMENT)
+KRATOS_CREATE_VARIABLE(double, FIBER_ORIENTATION_ANGLE)
```

Comment:
can't we deal with this by using the LOCAL_AXIS_* vectors? (i admit that the answer may be no. this is a real question)

Reply:
I think I can, I was planning to do this in the cleanup of the shells